modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js

```diff
@@ -120,9 +120,8 @@ export function populateConfigData([
 		tableView: tableViews.find(({id}) => id === app.dataListViewId),
 	};
 
-	if (app.active && config.formView.missingRequiredFields.nativeField) {
+	if (config.formView.missingRequiredFields.nativeField) {
 		app.dataLayoutId = null;
-		config.formView = {};
```

Comment:
@albinojunior we are doing this `app.dataLayoutId = null` to prevent the user from deploying the app, but if we clean the form view from config too, the user won't be able to see the warning icons across the app.

Reply:
Ok, makes sense.